src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/Contract.Service.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TestCategories>OuterLoop</TestCategories>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>11.0</MinimumVisualStudioVersion>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Contract.Service.Tests</RootNamespace>
+    <AssemblyName>Contract.Service.Tests</AssemblyName>
+    <SignAssembly>false</SignAssembly>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ProjectGuid>{8773ac5c-5b4d-4745-becf-01415cb10f1c}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="ServiceContractTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include='$(WcfSourceProj)'>
+      <Project>{9e50e7bf-cd6e-4269-a6dd-59fd0bd6c0fd}</Project>
+      <Name>System.Private.ServiceModel</Name>
+    </ProjectReference>
```

Comment:
Should read: 

```
    <!-- Compile tests against the contract, but copy our local-built implementation for testing -->
    <ProjectReference Include='$(WcfSourceProj)'>
      <Project>{9e50e7bf-cd6e-4269-a6dd-59fd0bd6c0fd}</Project>
      <Name>System.Private.ServiceModel</Name>
      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
      <OutputItemType>Content</OutputItemType>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
    </ProjectReference>
```

unless there's a really good reason for using the P2P references. Using the above ensures that the tests are testing only the surface area that is available in the public contract.